src/twisted/newsfragments/9410.bugfix

```diff
@@ -0,0 +1 @@
+twisted.web.http.Request no longer throws an Exception on write when in _disconnected state.
```

Comment:
Can probably just say "disconnected" state here.  It gets the idea across and the _ isn't clear to anyone not familiar with the private implementation details of the class (and may screw up the NEWS markup).

Reply:
language changed in https://github.com/twisted/twisted/pull/1106/commits/00a59794e8f3f6d9e596c3c600ef6c0dba3da084